### PR TITLE
Feature/reset commit

### PR DESCRIPTION
**Pull request by matthewpeterkort**

### CODE REVIEW

File: etl-job/fhir_import_export.py [hunk 1]
```diff
@@ -102,7 +105,7 @@ def _can_create(output: list[str],
     return can_create
 
 
-def _can_read(output: list[str],
+def _can_read(output: dict,
```
Comment (bwalsh) [inline etl-job/fhir_import_export.py:108]: Thanks for finding this bug

File: etl-job/fhir_import_export.py [hunk 2]
```diff
@@ -145,44 +148,54 @@ def _can_read(output: list[str],
...
-        output['logs'].append(f"ERROR UNZIPPING /tmp/{object_id}")
-        if result.stderr:
-            output['logs'].append(result.stderr.read().decode())
-        if result.stdout:
-            output['logs'].append(result.stdout.read().decode())
-        return False
 
-    output['logs'].append(f"UNZIPPED {file_path}")
-    return True
+def _download_and_unzip(object_id: str,
```
Comment (bwalsh) [inline etl-job/fhir_import_export.py:173]: Good idea separating concerns into two functions

**Review by bwalsh — COMMENTED** [review]
MP:  this all looks great.  I'm going to hold off approving (merge to development until we get some more testing)

**Comment by matthewpeterkort** [comment]
This concept got replaced with bundle concept

**Pull request closed without merging**